README.md

```diff
@@ -231,6 +231,8 @@ Take a look at the following overview of the Admin App extension samples we prov
 
 * [Custom list components](./src/tree#custom-list-components) - This extension is used to replace a part of each item shown in the tree used throughout the AdminApp. Currently supported only for the related data functionality.
 
+* [Custom notifications icons](./src/custom-notifications-icons) - This extension is used to generate css class that can be used to style the notification icon. If you have some custom notifications you can use this extension to create your own icon, isntead of using the default one. You can also use it to override the already available icons in the system.
```

Comment:
Lets make it clear that this is about system notifications and not app notifications (like the one that appears when you publish, etc.). Maybe add a link to the SF documentation as well

Reply:
We don't have a documentation link (it is not completed yet), but will log an action item to add it, when it is ready